subm/src/subm.js

```diff
@@ -104,24 +118,49 @@ const Site = freeze({
 
   testnetRoles: {
     'testnet-participant': '819067161371738173',
+    Community: '757035496198111262',
     // TODO: support for many-to-one discord to moniker
     // 'testnet-teammate': '825108158744756226',
     team: '754111409645682700',
   },
 
   badLogin: () => `${AgoricStyle.top}
-  <p><strong>Login refused.</strong> Only Incentivized Testnet participants are allowed.</p>
+  <p><strong>Login refused.</strong> Only ${keys(Site.testnetRoles).join(
```

Comment:
Nit: the keys of `testnetRoles` are not exactly the most human friendly.

Reply:
I'm not inspired to address that.